src/Uno.UI/UI/Xaml/Controls/Popup/Popup.md

```diff
@@ -0,0 +1,16 @@
+# Popup
+
+
+If you want to show a dimmed overlay underneath the popup, set the `Popup.LightDismissOverlayMode` property to `On`.
+
+If you wish to customise the overlay color, add the following to your top-level `App.Resources`:
+```xaml
+		<SolidColorBrush x:Key="PopupLightDismissOverlayBackground"
+						 Color="Red" />
+```
+
+
+
+
+
+			
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[code-block-style] Code blocks should be fenced](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5008853)